### DISCUSSION TITLE
The antivirus software reports a virus when packaging with PyInstaller 6.3.0.

### DISCUSSION THREAD

**Discussion by Lling00** [pyinstaller/pyinstaller#8207] [PyInstaller]
Currently, only PyInstaller 6.3.0 exhibits this issue; version 6.2 does not. Even if the antivirus software is turned off during packaging, the resulting executable still triggers a virus alert. We hope this issue can be resolved.

**Comment by bwoodsend** [comment]
Report the false positive to the antivirus software. We have no control over this.

**Reply by sanderjo** [reply]
I created a `hello_world.py`, and created an onefile .exe with  `python -m PyInstaller --onefile hello_world.py` both with pyinstaller 5.13.2 and 6.3.0. That first was OK according to Microsoft Defender on Virustotal, the last one was reported as having a virus.

https://www.virustotal.com/gui/file/59538ee311180e1a92ea36618fbe12c468476ad56776bee3a47857d6a288a1f0?nocache=1
https://www.virustotal.com/gui/file/6f88b8f3c8e88d5a6799de20f640835819c4c320164e1c896ead089f43e24957?nocache=1


So I reported it to Microsoft as a fake positive. Fingers crossed!


![image](https://github.com/pyinstaller/pyinstaller/assets/1273502/4cce3cd0-58ed-4ce3-8f03-051535ca007e)

**Comment by Safihre** [comment]
@bwoodsend @rokm 
As a developer, I would have also said that, but the fallout is larger than usual this time.
We just released a new version of SABnzbd 3 days ago and it's just been a constant stream of users complaining about Windows Defender blocking the executable, even removing it after the installation.
We always have problems with a few virusscanners, we even create a special FAQ for it: https://sabnzbd.org/wiki/faq#virusscanners
But this release it's just been crazy.

We already don't use `onefile`, but it still triggers on `SABnzbd.exe` (only 200kb), so basically on the PyInstaller bootloader.
Today I decided to downgrade to PyInstaller 5.13.2 and just rebuild the release with the older version. On virustotal the 5.13.2 version only has 2/69 hits, while the 6.3 release has 6/69 hits, including Defender.

Not sure how much you can do about it.. Probably not much, but it's still a real problem.

https://www.reddit.com/r/SABnzbd/comments/18xjfzc/release_notes_sabnzbd_420/
https://tweakers.net/downloads/67192/sabnzbd-421.html?showReaction=19485292#r_19485292
https://github.com/sabnzbd/sabnzbd/issues/2770
https://www.reddit.com/r/SABnzbd/comments/18zrb6a/version_421_18f4cc2_win_defender_alert/
https://www.reddit.com/r/usenet/comments/18xjfz7/comment/kg92p7j/?utm_source=reddit&utm_medium=web2x&context=3
